### ISSUE TITLE
I just get a loading spinner

### ISSUE THREAD

**Issue by samthomson** [gilbitron/Splashify#6]
Installed but on first run it just has a loading spinner. Any ideas? I'm on windows 

![image](https://cloud.githubusercontent.com/assets/6588340/21843769/76ea2572-d7ec-11e6-838f-2ea819062115.png)

I'm on windows ten

![image](https://cloud.githubusercontent.com/assets/6588340/21843806/9e5dc5aa-d7ec-11e6-8d06-cabda07d6d6b.png)

**Comment by jadchaar** [comment]
Same issue on OS X

**Comment by gilbitron** [comment]
Were you connected to internet? Did this only happen on the first run?

**Comment by samthomson** [comment]
Opened program again and it works. I was connected to the internet when I experienced the problem.

**Comment by gilbitron** [comment]
@samthomson Ok thanks for the feedback. I'll look into this.

**Comment by jadchaar** [comment]
@gilbitron was on internet. It just loaded when trying again. It was quite an odd bug I guess.

**Comment by PilloryBarnDesign** [comment]
Getting this issue, and also the red box saying it can't get the images. Tried connected to internet via Wifi and Ethernet and same problem.

Have quit multiple times and error persists.

Any ideas?

**Comment by gilbitron** [comment]
It looks like the issue is probably caused by us hitting the 500 requests/hour rate limit that Unsplash imposes on it's API. The best thing to do it try again after this hour finishes. I'll have a look into doing something to circumvent this.

**Comment by gilbitron** [comment]
I'm hoping this issue will have been fixed in v1.3.1. If anyone could confirm that would be great?

**Comment by PilloryBarnDesign** [comment]
Yep seems to be working :) Will let you know if I run into any trouble.

Thank you!

Andy
